pkg/apis/pipeline/v1/when_types.go

```diff
@@ -27,15 +27,21 @@ import (
 // to determine whether the Task should be executed or skipped
 type WhenExpression struct {
 	// Input is the string for guard checking which can be a static input or an output from a parent Task
-	Input string `json:"input"`
+	Input string `json:"input,omitempty"`
```

Comment:
Not sure if we should add it in this PR, since the functionality of `CEL` is not implemented yet. I'm ok with adding it later but open to comments here

Reply:
It's okay to add it, so that this commit contains the API changes